src/class.js

```diff
@@ -0,0 +1,23 @@
+// eslint-disable-next-line
```

Comment:
- [ ] Please do not abuse linter disabling, currently, you have disabled the linters 10 times linting is important to reduce errors and improve the overall quality of your code. Using lint tools can help you accelerate development and reduce costs by finding errors earlier, but you will lose all these advantages if you disable them.
    - I will kindly ask you to fix the most linter errors you can instead of disabling them, than you 😄